analysis/model/04_01_(b)_cox_format_survival_data.R

```diff
@@ -8,24 +8,22 @@ fit_get_data_surv <- function(event,subgroup, stratify_by_subgroup, stratify_by,
   #------------------ RANDOM SAMPLE NON-CASES for IP WEIGHING ------------------
   set.seed(137)
   
-  cases <- survival_data %>% filter((!is.na(event_date)) & 
-                                      (
-                                        event_date == follow_up_end
-                                      ))
-  
-
   print(paste0("Total number in survival data: ", nrow(survival_data)))
-  print(paste0("Number of cases: ", nrow(cases)))
   
-  if(subgroup != "covid_pheno_hospitalised"){
+  if(nrow(survival_data) >= 4000000){
```

Comment:
extremely extremely pedantic, but the protocol and print statement here just says "more than" so I don't think the ">=" is needed, just a ">" before the 4000000 so it matches what the wording says.